### ISSUE TITLE
BUG: Error: rpc error: code = Unknown desc = rpc error: code = Unknown desc = failed to execute message; message index: 0: account paloma1... already exists: invalid request

### ISSUE THREAD

**Issue by taariq** [VolumeFi/paloma#1437]
Attempting to create vesting accounts fail with `account already exists error`. 

Command Usage:  `palomad tx vesting create-periodic-vesting-account [to_address] [periods_json_file] [flags]`

Error: rpc error: code = Unknown desc = rpc error: code = Unknown desc = failed to execute message; message index: 0: account paloma102rffmh8u0xk8h4zkulq8shwgjvjxxvzee84n3 already exists: invalid request [cosmos/cosmos-sdk@v0.50.5/x/auth/vesting/msg_server.go:192] with gas used: '54241': unknown request

1. After sending Tx: 
failed to execute message; message index: 0: account paloma102rffmh8u0xk8h4zkulq8shwgjvjxxvzee84n3 already exists: invalid request

2. Tx Hash https://paloma.explorers.guru/transaction/7A95006D74DEECEF6CE069962CDAC690013F1F17D9041D82FD6E4B08692E1F93

3. There is a balance on the account: https://paloma.explorers.guru/account/paloma102rffmh8u0xk8h4zkulq8shwgjvjxxvzee84n3

4. When testing the command with a 0 balance account, the error is different.
palomad tx vesting create-periodic-vesting-account paloma1nvsulw0wvfsc73dt5jm6mpytlcah5pqkyuhn2t ./paloma1nvsulw0wvfsc73dt5jm6mpytlcah5pqkyuhn2t.json --from paloma1.. --node $NODE --chain-id messenger --fees 400ugrain --gas auto -b sync --yes

returns the error:

Error: rpc error: code = Unknown desc = rpc error: code = Unknown desc = failed to execute message; message index: 0: spendable balance 0grain is smaller than 10000000grain: insufficient funds [cosmos/cosmos-sdk@v0.50.5/x/bank/keeper/send.go:278] with gas used: '73502': unknown request.

1. The denom looks off. we use ugrain, but now we have grain in the error?
2. The balance of zero is insufficient for the creation of the account? the from address is a different address.
3. It seems the error is with the command, not the balance?

**Comment by byte-bandit** [comment]
@taariq So the answer is very simple, but I had to actually look at the code implementation in Cosmos because it is written NOWHERE explicitly:

You **cannot** vest into an already existing account. This means, if you'd like to vest GRAINS for everyone, you will need them to create a fresh key and share the address with you. It cannot exist yet.

I've successfully vested (tx hash `88BE1624EEE953DA209FCCFA50A40BCE1EF1B2B0EF06D83CA37863A28E3E323C`) against a new account I created from which I can now withdraw what has already been deposited.
